controllers/apply/form-router-next/steps.js

```diff
@@ -109,9 +110,13 @@ module.exports = function(formId, formBuilder) {
                 currentApplicationData
             } = res.locals;
 
+            const sanitisedBody = sanitiseRequestBody(req.body);
+
+            console.log(sanitisedBody);
```

Comment:
❌

Reply:
(though wonder if we should log the number of sanitisations made?)

Reply:
🙈 And good call, added debug level logging to the sanitise method:

![image](https://user-images.githubusercontent.com/123386/61627618-26e36300-ac78-11e9-9515-2f7b7fa32c26.png)